src/lib/y2storage/storage_manager.rb

```diff
@@ -281,20 +303,14 @@ def fake_from_yaml(yaml_file = nil)
       #
       # @return [StorageManager] singleton instance
       def fake_from_xml(xml_file)
```

Comment:
We maybe should rethink these methods a little bit. The were creating (literally from documentation) as "an alternative to the instance method"... when the instance method implied probed. Now that probing and creating the instance are different things, maybe we should reconsider how they work.